Sources/GeoJSONSwiftHelper/GeoJSONSwiftHelper.swift

```diff
@@ -52,30 +50,41 @@ extension GeoJSONObject {
   public var polygons: [Turf.Polygon] {
     switch self {
     case .geometry(let geometry):
-      return getPolygons(from: geometry)
+      return geometry.polygons
+    case .feature(let feature):
+      return feature.geometry!.polygons
```

Comment:
Should try not to force unwrap... you don't want crashes in case the geoJSON doesn't have a geometry.

```suggestion
      return feature.geometry?.polygons ?? []
```

Reply:
Or return optional